go/app/main.go

```diff
@@ -28,14 +43,113 @@ func root(c echo.Context) error {
 func addItem(c echo.Context) error {
 	// Get form data
 	name := c.FormValue("name")
+	category := c.FormValue("category")
+	image := c.FormValue("image")
```

Comment:
This may work because `curl` and the Go API are on the same computer.

When we put the Go API on a server on the internet, then the `curl` command does not run on the same computer. So our Go API will not be able to save the file. 😿 

I will post some resources on Slack so you can have more examples.